docs/flashbots-mev-boost/block builders.md

```diff
@@ -1,51 +1,75 @@
-- `submitBlock`: [https://builder-relay-goerli.flashbots.net/relay/v1/builder/blocks](https://builder-relay-goerli.flashbots.net/relay/v1/builder/blocks) (POST)
+Various methods for defining block value were [considered](https://collective.flashbots.net/t/block-scoring-for-mev-boost-relays/202) by members of the community. It was determined that block level scoring was the most simple and intuitive method for scoring block value.
 
-**Sepolia relay block submission endpoints:**
 
-- `getValidators`: [https://builder-relay-sepolia.flashbots.net/relay/v1/builder/validators](https://builder-relay-sepolia.flashbots.net/relay/v1/builder/validators) (GET) - array of proposer preferences for registered validators with assigned duties in the current and next epoch
-- `submitBlock`: [https://builder-relay-sepolia.flashbots.net/relay/v1/builder/blocks](https://builder-relay-sepolia.flashbots.net/relay/v1/builder/blocks) (POST)
+### Block level scoring
 
-**Rate-limits:** 
+Block level scoring looks at the difference in the balance of the fee recipient account before and after the block execution.  
 
-Submissions to all relays are currently rate-limited to 60 / minute.
+Note that a “block score” is not meant to be a formal definition of realized extractable value, since this is a difficult metric to quantify. For example, a Layer 2 transfer to a validator’ fee recipient address could be considered extractable value, but falls outside the scope of a block score calculation. 
 
-**See also:**
+Constructing a payment proof for this scoring method requires a Merkle Proof of the fee recipient balance in block *(n - 1)*, and a Merkle Proof of the fee recipient balance in block *n*. **Payment proofs have not yet been put into production.** Active discussion about payment proof implementation is still on-going. For more details or to participate in the discussion around payment proofs and block-level scoring, please check out to the [block scoring](https://collective.flashbots.net/t/block-scoring-for-mev-boost-relays/202) forum thread.
 
-- [MEV-Boost Geth Builder](https://github.com/flashbots/boost-geth-builder) - an example builder implementation
-- [Relay API documentation - Block Builder API](https://bit.ly/3BmGZ3T)
-- Block Builder Self-Help Group: [https://collective.flashbots.net/c/builders/14](https://collective.flashbots.net/c/builders/14)
-- Github issue about becoming block builder: [https://github.com/flashbots/mev-boost/issues/145](https://github.com/flashbots/mev-boost/issues/145)
+## External Builders
+
+External builders can submit blocks to Mainnet, Goerli and Sepolia Flashbots relays. The table below outlines Builder API methods available on each network.
+
+### Relay Block Submission Endpoints by Network
+
+|  |  | Mainnet | Goerli | Sepolia |
```

Comment:
may be better to invert the table -- to have the networks as rows, and the APIs as columns?